javascript/javascript1/week2/studentManager.js

```diff
@@ -0,0 +1,62 @@
+const class07Students = [];
+function addStudentToClass(studentName) 
+{    
+    if (typeof(studentName) == "string")
+   
+ {        
+    if (studentName == "Queen" || class07Students.length < 6) 
+    {
+        return class07Students.push(studentName);    
+        } 
+        
+        
+        else if (class07Students.includes(studentName)) {
```

Comment:
I believe with your current _if_ logic you can still add two students with the same name. Can you try adding two students with the same name and see that ?

Reply:
no i tried it doesnt allow same name